dask/array/reshape.py

```diff
@@ -248,48 +244,6 @@ def reshape(x, shape, merge_chunks=True, limit=None):
         x = x.rechunk({i: 1 for i in range(din - dout)})
 
     inchunks, outchunks = reshape_rechunk(x.shape, shape, x.chunks)
-    # Check output chunks are not too large
-    max_chunksize_in_bytes = reduce(mul, [max(i) for i in outchunks]) * x.dtype.itemsize
-
-    if limit is None:
-        limit = parse_bytes(config.get("array.chunk-size"))
-        split = config.get("array.slicing.split-large-chunks", None)
```

Comment:
is this config still in use?

Reply:
if it is no longer in use (and you do not intend to reintroduce it) we should raise a warning that this is now being ignored and remove it from the dask.yaml

Reply:
take was the other case where this option was used, but I ripped this out as well with the new shuffle backend that takes care of this automatically.

I'll open a follow up to add a DeprecationWarning